bundles/org.eclipse.ui.ide/src/org/eclipse/ui/ide/fileSystem/zip/ZipFileSystemContributor.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
```

Comment:
2024 Vector?

Reply:
This method is copied unchanged from the example project. We thought this would be the best place for it but did not change it, thus the Copyright is still the old one.